app/models/intake.rb

```diff
@@ -170,9 +166,6 @@
 #  created_at                                           :datetime
 #  updated_at                                           :datetime
 #  client_id                                            :bigint
-#  intake_ticket_id                                     :bigint
-#  intake_ticket_requester_id                           :bigint
-#  primary_intake_id                                    :integer
```

Comment:
I'm assuming you've checked that `primary_intake_id` is not in use anywhere.

Reply:
```

Reply:
```
$ git grep -l primary_intake_id
db/migrate/20200612170554_add_primary_intake_id_to_intake.rb
db/migrate/20210218014835_delete_more_zendesk_columns.rb
$
```

once added, then once
tossed away. our friend, column:
primary intake